pkg/controller/provider/model/vsphere/model.go

```diff
@@ -141,6 +153,7 @@ type Host struct {
 	Datastores         []Ref          `sql:""`
 	HostScsiDisks      []HostScsiDisk `sql:""`
 	AdvancedOptions    Ref            `sql:""`
+	StorageProtocols   []ProtocolType `sql:""`
```

Comment:
I am unsure if this could be just array of protocol types, because we need to be able to map it later to specific storage ( I think via naa)

Reply:
Added more information, please take a look